docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,103 @@
 ---
-title: Machine learning data transforms - ML.NET
+title: Data transformations | ML.NET
 description: Explore the feature engineering components supported in ML.NET.
-author: JRAlexander
-ms.custom: seodec18
-ms.date: 01/14/2019
+author: nake
```

Comment:
If using your GitHub username, I think you missed the t.

Reply:
Whoops! Who is this nake?